sympy/functions/elementary/tests/test_exponential.py

```diff
@@ -117,6 +117,9 @@ def test_exp_rewrite():
     assert exp(x).rewrite(tanh) == (1 + tanh(x/2))/(1 - tanh(x/2))
     assert exp(pi*I/4).rewrite(sqrt) == sqrt(2)/2 + sqrt(2)*I/2
     assert exp(pi*I/3).rewrite(sqrt) == S(1)/2 + sqrt(3)*I/2
+    assert exp(x*log(y)).rewrite(Pow) == y**x
+    assert exp(log(x)*log(y)).rewrite(Pow) == x**log(y)
```

Comment:
Should this be `assert exp(log(x)*log(y)).rewrite(Pow) in [x**log(y), y**log(x)]`?